src/service/video-manager-impl.js

```diff
@@ -436,6 +540,136 @@ class VideoEntry {
   }
 
   /**
+   * Called when the video's position in the viewport changed.
+   * @param {PositionInViewportEntryDef} newPos
+   * @private
+   */
+  onPositionChanged_(newPos) {
+
+    const vidElement = scopedQuerySelector(this.video.element, 'video, iframe');
```

Comment:
let's only query once. ( can do it in `videoLoaded_` and cache it in an instance variable like `this.videoOrIFrameElem_`